rust/src/go/nmstate/nmstate.go

```diff
@@ -0,0 +1,199 @@
+// +build cgo
+package nmstate
+
+// #cgo CFLAGS: -g -Wall
+// #cgo LDFLAGS: -lnmstate
+// #include <nmstate.h>
+// #include <stdlib.h>
+import "C"
+import (
+	"fmt"
+	"io"
+	"time"
+)
+
+type Nmstate struct {
+	Timeout	    uint
+	LogsWritter io.Writer
+	Flags	    byte
+	State	    string
+	Checkpoint  string
+}
+
+const (
+	none byte = 0 << iota
+	kernel_only
+	no_verify
+	include_status_data
+	include_secrets
+	no_commit
+)
```

Comment:
```suggestion
const none = 0
const (
	kernelOnly = 1 << iota
	noVerify
	includeStatusData
	includeSecrets
	noCommit
)
```

Reply:
I noticed that `none` is not needed. I changed the other flags names to use the correct format.